factory-package-news/factory-package-news.py

```diff
@@ -18,6 +18,15 @@
 changelog_max_lines = 100  # maximum number of changelog lines per package
 
 
+# rpm's python bindings changed in version 4.15 [0] so that they actually return
+# utf-8 strings. Leap 15 ships an older version, so this is needed to
+# keep this script working there too.
+#
+# [0] https://github.com/rpm-software-management/rpm/commit/84920f898315d09a57a3f1067433eaeb7de5e830
+def forcestr(content):
```

Comment:
I wonder if `utf8str` would be a slightly besser name.

Reply:
Thanks, yeah definitely looks better than `forcestr` :) Should I make another PR that changes that?

Reply:
Yes please if you agree